builder/chroot-script.sh

```diff
@@ -220,3 +225,13 @@ rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 echo "HYPRIOT_DEVICE=\"$HYPRIOT_DEVICE\"" >> /etc/os-release
 echo "HYPRIOT_IMAGE_VERSION=\"$HYPRIOT_IMAGE_VERSION\"" >> /etc/os-release
 cp /etc/os-release /boot/os-release
+
+# install Witty pi energy manager software
+echo "Installing i2c-tools"
+apt-get install i2c-tools
+mkdir /home/imvec
+cd /home/imvec
+echo "Installing Witty Pi 2 software"
+wget http://www.uugear.com/repo/WittyPi2/installWittyPi.sh
+sh installWittyPi.sh
+apt-get update
```

Comment:
This shouldn't be necessary (it will fetch package lists once again after cleanup). If apt is needed perhaps we should move apt cleanup further down :-)